recipes-kernel/linux/files/prplmesh.cfg

```diff
@@ -0,0 +1,28 @@
+CONFIG_TUN=y
+CONFIG_DUMMY=y
+CONFIG_BRIDGE=y
+CONFIG_BRIDGE_NETFILTER=y
+CONFIG_BRIDGE_NF_EBTABLES=m
+CONFIG_BRIDGE_EBT_BROUTE=m
+CONFIG_BRIDGE_EBT_T_FILTER=m
```

Comment:
These two are sufficient for the ebtables that we use. (Actually, only one of them, but we may switch to a different one in the future so better keep both).

Or does the ebtables program barf if not all of them are available?

Reply:
No,
I've checked that kernel menuconfig allows to keep enabled only options mentioned by you among another ebtables modules
So i'll keep only these two options and remove another one to allow save current values